src/main/java/r4v88/api/AuthorService.java

```diff
@@ -16,6 +16,6 @@
 
     void addAuthor(Author author);
 
-    void removeAuthorById(Long id);
+    void removeAuthorById(long id);
```

Comment:
Z ciekawostek - w springu data typ prosty jako id by nie przeszedl. Nie wymaga to zmiany po twojej stronie :)

Reply:
czyli wszystkie typy musza byc obiektowe, dobrze wiedziec bo mam nadzieje ze niedlugo sie za Springa wezme ;)?